db/controller/topics.controller.js

```diff
@@ -125,3 +126,19 @@ exports.patchArticle = async (req, res, next) => {
 };
 
 
+exports.deleteComment = async (req, res, next) => {
+  try {
+    const { comment_id } = req.params;
+    if (isNaN(comment_id)) {
+      return res.status(400).send({ message: "bad request" });
+    }
```

Comment:
I like the idea of this, but this doesnt need to have a separate check like this as allowing this to reach the sql query will result in an sql error for you. 

We also shouldnt have error handling in the controller file and should handle error throwing in the model